website/docs/exempt-namespaces.md

```diff
@@ -3,6 +3,8 @@ id: exempt-namespaces
 title: Exempting Namespaces
 ---
 
+`Feature State`: The `Config` resource is currently alpha.
```

Comment:
can we add this in https://open-policy-agent.github.io/gatekeeper/website/docs/sync too?